frontend/src/pages/Alerts/AlertForm/index.tsx

```diff
@@ -256,6 +298,31 @@ export const AlertForm: React.FC = () => {
 		navigate(`/${projectId}/alerts`)
 	}
 
+	const settings: AlertSettings = {
+		productType,
+		functionType,
+		functionColumn,
+		query,
+		alertName,
+		groupByEnabled,
+		groupByKey,
+		thresholdValue,
+		thresholdCondition,
+		thresholdType,
+		thresholdWindow,
+		thresholdCooldown,
+		destinations,
+	}
+
+	const settingsEncoded = btoa(JSON.stringify(settings))
```

Comment:
Nit, but would it be more performant to only create the settings + do the encoding inside the `useEffect`? Or do we need this as a dependency to trigger the logic?